agent.go

```diff
@@ -224,6 +231,47 @@ func (s *sandbox) readStdio(cid string, pid int, length int, stdout bool) ([]byt
 	return buf, nil
 }
 
+func (s *sandbox) setupSharedPidNs() error {
+	cmd := exec.Command(selfBinPath, pauseBinArg)
```

Comment:
please keep the arg[0] unchanged via using
```go
&exec.Cmd{
     Path: selfBinPath, 
     Args: []string{os.Args[0], pauseBinArg},
}

Reply:
And please document that.